src/terms/local.jl

```diff
@@ -95,7 +95,7 @@ end
     ρ_fourier = r_to_G(basis, total_density(ρ))
 
     # energy = sum of form_factor(G) * struct_factor(G) * rho(G)
-    # where struct_factor(G) = cis(-2π G⋅r)
+    # where struct_factor(G) = e^{-i Gr}
```

Comment:
Why? The original was correct, no?

Reply:
Oh, I see. I would keep the `\cdot` and than say explicitly that `r` and `G` are in cartesian coordinates (because usually we use `G` with fractional).

Reply:
`G` is in Cartesian for `compute_forces` but not always for `AtomicLocal`. (https://github.com/JuliaMolSim/DFTK.jl/pull/662/commits/b9f1432cb51ead3a641ee8ad1f1319bd314fb9d9)

Reply:
Nah we don't follow that consistently any more.